oci/runtime_oci.go

```diff
@@ -280,6 +280,7 @@ func (r *runtimeOCI) ExecContainer(c *Container, cmd []string, stdin io.Reader,
 	args = append(args, "--process", processFile.Name())
 	args = append(args, c.ID())
 	execCmd := exec.Command(r.path, args...)
+	execCmd.Env = append(execCmd.Env, fmt.Sprintf("XDG_RUNTIME_DIR=%s", os.Getenv("XDG_RUNTIME_DIR")))
```

Comment:
What happens if XDG_RUNTIME_DIR is not defined.  Will commands work with it set to 
""?

Reply:
it depends on the program whether they are checking for the variable existence or if it has a value.  Usually it is checked whether it has a value != "".  `os.Getenv` considers them the same, `os.LookupEnv` must be used to check if it is set.

runc when running in a namespace expects this to be set otherwise it will use `/run/runc` which is not accessible to the rootless user.

Reply:
changed to use `LookupEnv`